src/main/java/org/broadinstitute/hellbender/utils/hmm/interfaces/CallStringProvider.java

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.hellbender.utils.hmm.interfaces;
+
+/**
+ * An interface for hidden states that provide a call string (i.e. a unique string identifier)
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public interface CallStringProvider {
```

Comment:
Since there is only one method, give this the `@FunctionalInterface` annotation.